src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -64,55 +63,47 @@
-                                        <label class="slds-form-element__label" for="totalMatchCheckbox">
-                                            {!v.batchMetadata.labels.totalMatchLabel}
-                                        </label>
-                                        <lightning:helptext content="{!v.batchMetadata.labels.totalMatchHelp}" />
-                                    </div>
-                                </aura:set>
-                            </aura:if>
+                            <div>
+                                <lightning:input aura:id="expectedCount" type="number"
+                                                 class="slds-p-top_small"
+                                                 min="0"
+                                                 label="{!v.wizardMetadata.labels.expectedCountLabel}"
+                                                 name="expectedCount"
+                                                 value="{!v.batchInfo.expectedCount}"/>
+                                <lightning:input aura:id="expectedTotal" type="number"
+                                                 formatter="currency"
+                                                 class="slds-p-top_small"
+                                                 min="0"
+                                                 step="0.01"
+                                                 label="{!v.wizardMetadata.labels.expectedTotalLabel}"
+                                                 name="expectedTotal"
+                                                 value="{!v.batchInfo.expectedTotal}"/>
+                                <lightning:input aura:id="totalMatchCheckbox" type="checkbox"
+                                                 class="bgeInputCheckbox slds-p-top_small"
+                                                 label="{!v.wizardMetadata.labels.totalMatchLabel}"
+                                                 name="requireTotalMatch"
+                                                 checked="{!v.batchInfo.requireTotalMatch}"
+                                                 variant="label-hidden"/>
+                                <label class="slds-form-element__label" for="totalMatchCheckbox">
+                                    {!v.wizardMetadata.labels.totalMatchLabel}
+                                </label>
+                                <lightning:helptext content="{!v.wizardMetadata.labels.totalMatchHelp}" />
+                            </div>
                         </lightning:layoutItem>
                     </lightning:layout>
                 </aura:if>
                 <!--End Step 0-->
 
                 <!-- Step 1: Select Fields -->
-                <div class="{!if(v.batchMetadata.progressIndicatorStep == '1', '', 'slds-hide')}">
+                <div class="{!if(v.wizardMetadata.progressIndicatorStep == '1', '', 'slds-hide')}">
```

Comment:
@randi274  Not to do with this PR's changes, but is there a reason we use a div with dynamic css on this step and aura:if on the other steps, to control visibility?

Reply:
We had done this originally because of the other component that wasn't working with a dynamic aura:if... As I recall it was the lightning:accordion or lightning:accordionSection? This is outdated now though. We can probably change this, but would require a little testing to be sure.